styles/themes.scss

```diff
@@ -2,6 +2,9 @@
 @import './themes/dark';
 
 .yc-root {
+    --dc-text-highlight: var(--yc-color-base-warning-heavy);
+    --dc-text-highlight-selected: #ffab3b;
```

Comment:
no base color for this?

Reply:
there is, but it is lighter in the dark theme than the browser highlight color
I'd set the browser color for both themes
Below you can see the insufficient contrast between highlighted elements and the current element

"much" is highlighted in the browser color, "resources" is highlighted in the base color
there is no difference in the light theme

<img width="715" alt="Screen Shot 2022-01-19 at 6 46 18 AM" src="https://user-images.githubusercontent.com/62902338/150060519-1a24b387-8d4e-4c3a-b328-155e626b7e85.png">

<img width="750" alt="Screen Shot 2022-01-19 at 6 52 47 AM" src="https://user-images.githubusercontent.com/62902338/150060928-f00ff28b-70f8-47cc-ad13-3444306a93e9.png">